bigframes/core/compile/compiled.py

```diff
@@ -163,6 +169,54 @@ def get_column_type(self, key: str) -> bigframes.dtypes.Dtype:
             bigframes.dtypes.ibis_dtype_to_bigframes_dtype(ibis_type),
         )
 
+    def _aggregate_helper(
```

Comment:
Could we get a better name for this method, please. "helper" is very generic, so it's hard to understand what this method is doing.

Reply:
Renamed it into `_aggergate_base` to match the object name `BaseIbisIR`.